tests/acceptance/features/apiGraphUserGroup/getUser.feature

```diff
@@ -1443,3 +1443,171 @@ Feature: get users
       | user                  | errorToken       |
       | Alice-From-Wonderland | -From-Wonderland |
       | Alice@From@Wonderland | @From@Wonderland |
+
+  @issue-7990
+  Scenario: non-admin user searches other users by e-mail
+    When user "Brian" searches for user "%22alice@example.org%22" using Graph API
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+    """
+    { 
+      "type": "object",
+      "required": [
+        "value"
+      ],
+      "properties": {
+        "value": {
+          "type": "array",
+          "required": [
+            "displayName",
+            "id",
+            "mail",
+            "userType"
+          ],
+          "properties": {
+            "displayName": {
+              "type": "string",
+              "enum": ["Alice Hansen"]
+            },
+            "id": {
+              "type": "string",
+              "pattern": "^%user_id_pattern%$"
+            },
+            "mail": {
+              "type": "string",
+              "enum": ["alice@example.org"]
+            },
+            "userType": {
+              "type": "string",
+              "enum": ["Member"]
+            }
+          }
+        }
+      }
+    }
+    """
+
+  @issue-7990
+  Scenario: non-admin user searches for a disabled users
```

Comment:
so searching disabled users is possible

CC @ScharfViktor

Reply:
Yes, it possible. old `sharees?search=mar` can also find disabled user
we forbid sharing to disabled user. I think the `@problem-7990` link here is incorrect

Reply:
the explanation here https://github.com/owncloud/ocis/issues/7962#issuecomment-1883179994 says that the sharing with disabled user is possible. And we added tests here https://github.com/owncloud/ocis/pull/8139